src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
https://github.com/JasonTaranenko/jv-oop/blob/homeWork/checklist.md#please-dont-add-redundant-empty-lines-to-your-code